source/_docs/pantheon-yml.md

```diff
@@ -36,6 +36,27 @@ protected_web_paths:
 * You may not be able to protect files or paths with special characters
 * Wait a few seconds for changes to take effect
 
+### HTTPS Redirects with HSTS Headers
+Ensure that your site will always use HTTPS to deliver content with `enforce_https`. Three values are available:
+
+- `off` (default: no https redirect)
+- `transitional` (Redirects to https and includes an HSTS header with a 5-minute duration)
+- `transitional+subdomains` (Redirects to https and includes an HSTS header with a 5-minute duration, including subdomains)
+- `full` (Redirects to https and includes an HSTS header with a 366-day duration)
+- `full+subdomains` (Redirects to https and includes an HSTS header with a 366-day duration, including subdomains)
+
+The preferred `hsts` configuration is shown below:
```

Comment:
Maybe somewhere around here we should put in a warning that `+subdomains` should not be used if the customer has any subdomains that are not hosted on Pantheon, unless they are sure they will behave correctly with these hsts settings.

Reply:
added below the "preferred hsts config"!